Unable to invoke method 'activate' for class org.apache.sling.xss.impl.XSSAPIImpl

When upgrading to the latest dependencies, I get following exception. Did this occur for you and did you do anything to fix it? I upgraded all other maven dependencies to the latest versions, but it doesn't seem to fix the problem:

```
java.lang.RuntimeException: Unable to invoke method 'activate' for class org.apache.sling.xss.impl.XSSAPIImpl

	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.invokeMethod(OsgiServiceUtil.java:366)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.lambda$invokeLifecycleMethod$0(OsgiServiceUtil.java:264)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.findAndInvokeNearestMethod(OsgiServiceUtil.java:150)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.invokeLifecycleMethod(OsgiServiceUtil.java:182)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.activateDeactivate(OsgiServiceUtil.java:98)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.activateInjectServices(OsgiServiceUtil.java:489)
	at org.apache.sling.testing.mock.osgi.MockOsgi.registerInjectActivateService(MockOsgi.java:268)
	at org.apache.sling.testing.mock.osgi.context.OsgiContextImpl.registerInjectActivateService(OsgiContextImpl.java:191)
	at org.apache.sling.testing.mock.osgi.context.OsgiContextImpl.registerInjectActivateService(OsgiContextImpl.java:179)
	at org.apache.sling.testing.mock.sling.context.SlingContextImpl.registerDefaultServices(SlingContextImpl.java:250)
	at io.wcm.testing.mock.aem.context.AemContextImpl.registerDefaultServices(AemContextImpl.java:74)
	at org.apache.sling.testing.mock.sling.context.SlingContextImpl.setUp(SlingContextImpl.java:195)
	at io.wcm.testing.mock.aem.context.AemContextImpl.setUp(AemContextImpl.java:108)
	at io.wcm.testing.mock.aem.junit5.AemContext.setUpContext(AemContext.java:100)
	at io.wcm.testing.mock.aem.junit5.AemContextExtension.setAemContextInStore(AemContextExtension.java:73)
	at io.wcm.testing.mock.aem.junit5.AemContextExtension.postProcessTestInstance(AemContextExtension.java:63)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.forEachRemaining(StreamSpliterators.java:312)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:735)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at java.base/java.util.Optional.orElseGet(Optional.java:369)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
Caused by: java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    jakarta/json/spi/JsonProvider$Holder.<clinit>()V @7: putstatic
  Reason:
    Type 'org/apache/johnzon/core/JsonProviderImpl' (current frame, stack[0]) is not assignable to 'jakarta/json/spi/JsonProvider'
  Current Frame:
    bci: @7
    flags: { }
    locals: { }
    stack: { 'org/apache/johnzon/core/JsonProviderImpl' }
  Bytecode:
    0000000: bb00 0359 b700 04b3 0001 b1            

	at jakarta.json.spi.JsonProvider.provider(JsonProvider.java:64)
	at jakarta.json.Json.createReaderFactory(Json.java:218)
	at org.apache.sling.xss.impl.XSSAPIImpl.activate(XSSAPIImpl.java:82)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.sling.testing.mock.osgi.OsgiServiceUtil.invokeMethod(OsgiServiceUtil.java:357)
	... 27 more
```

--- a/README.md
+++ b/README.md
@@ -30,6 +30,9 @@ To import the dependencies in your AEM project use:
 </dependency>
 ```
 
+*Please note*: Since version `2024.6.16971.20240628T122619Z-240600.0001` JSONP 2.1 and Johnzon 2.x is used by default by this POM. If your code, or dependencies of your project, still rely on JSONP 1.1 and Johnzon 1.2.x, you can use the [Mixin for JSONP 1.1](https://github.com/wcm-io/io.wcm.maven.aem-cloud-dependencies-mixin-jsonp11).
+
+
 
 ## Release cycle
 

--- a/pom.xml
+++ b/pom.xml
@@ -529,15 +529,21 @@
       <dependency>
         <groupId>org.apache.johnzon</groupId>
         <artifactId>johnzon-core</artifactId>
-        <!-- Stick with Johnzon 1.x as long as other modules (esp. Adobe Core Components) still depend on javax.json -->
-        <!-- update-aem-deps:derived-from=com.adobe.granite.commons.johnzon:1.2.16 -->
-        <version>1.2.21</version>
+        <!-- update-aem-deps:derived-from=org.apache.sling.commons.johnzon:2.0.0 -->
+        <version>2.0.0</version>
       </dependency>
       <dependency>
-        <groupId>org.apache.geronimo.specs</groupId>
-        <artifactId>geronimo-json_1.1_spec</artifactId>
-        <!-- update-aem-deps:derived-from=com.adobe.granite.commons.johnzon:1.2.16 -->
-        <version>1.3</version>
+        <groupId>jakarta.json</groupId>
+        <artifactId>jakarta.json-api</artifactId>
+        <!-- update-aem-deps:derived-from=org.apache.sling.commons.johnzon:2.0.0 -->
+        <version>2.1.1</version>
+      </dependency>
+      <!-- Update to latest Sling JSON Content Parser 2.x for unit tests for compatibility with Johnzon 2.0 (switch from javax.json to jakarta.json) -->
+      <dependency>
+        <groupId>org.apache.sling</groupId>
+        <artifactId>org.apache.sling.contentparser.json</artifactId>
+        <!-- update-aem-deps:ignore -->
+        <version>2.1.0</version>
       </dependency>
       <dependency>
         <groupId>org.apache.sling</groupId>

